tasks/README.md

```diff
@@ -6,4 +6,11 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task305_QUAREL_correct_answer_generation` | Given a sentence and a question, write the correct answer based on the sentence. | Answer Generation
```

Comment:
Also lower case the filenames and also in the name column in README. `task305_QUAREL_correct_answer_generation` to task305_quarel_correct_answer_generation

Reply:
done

Reply:
<img width="850" alt="Screen Shot 2021-09-14 at 7 34 52 PM" src="https://user-images.githubusercontent.com/74945781/133361019-bbe7ad9e-2545-4bc5-bf12-8a225c2fc2a0.png">

Hi, I've renamed the filenames, but it showed "all checks have failed" when I committed the changes. Does it have any impact on the changes?